src/org/omegat/core/segmentation/SRX.java

```diff
@@ -110,40 +105,105 @@ public SRX copy() {
     }
 
     /**
-     * Saves segmentation rules into specified file.
+     * Saves segmentation rules into specified directory.
+     * Possibilities:
+     * -- if file does not exist : SRX format
+     * -- if conf file or srx file exists: owerwrite it
```

Comment:
Point 1: see the answer I posted in Preferences.java --> yes, the comment is obsolete, now it can only save to SRX.

Point 3: as the lines saying "if (srx == null) outFile.delete()" are not written by me, I would like that we check together if the situation can really happen, or if it is just an old piece of code which is not valid anymore. Then we can check whenever we should comment this case or not.

Reply:
Corrected